hack/make-rules/test.sh

```diff
@@ -263,6 +265,7 @@ runTests() {
           ${testargs[@]:+${testargs[@]}} \
         | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
         | grep \"${go_test_grep_pattern}\"" \
+          {} \
```

Comment:
There's also a mix of 2 and 4 space indentations going on here and elsewhere in the file. Which is the preferred style?